controller/handler/trial.js

```diff
@@ -124,6 +127,10 @@ function trialView (request, reply) {
 
             const endDate = processRules(ruleValues, Date.now());
 
+            if (request.query.requestType === 'ajaxRequest') {
```

Comment:
First when the control enters `trial.js` for the first time, we are returning back the whole page but when it enters when an ajax request is sent then at that time we are returning just an array of new compliance values. In order to distinguish between these two `request`and`reply ` I gave  that if condition